<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingForm</class>
 <widget class="QDialog" name="TrainingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add training departure</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="lbDepType">
           <property name="text">
            <string>Departure type</string>
           </property>
           <property name="buddy">
            <cstring>lineEditDepType</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditDepType">
           <property name="text">
            <string>тренировка</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="lbTrainingType">
           <property name="text">
            <string>Training type</string>
           </property>
           <property name="buddy">
            <cstring>comboBoxTrainingType</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxTrainingType"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="lbAdress">
           <property name="text">
            <string>Address</string>
           </property>
           <property name="buddy">
            <cstring>lineEditAddress</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditAddress"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="lbDepDate">
           <property name="text">
            <string>Departure date</string>
           </property>
           <property name="buddy">
            <cstring>dateEditDeparture</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditDeparture"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="lbDepTime">
           <property name="text">
            <string>Departure time</string>
           </property>
           <property name="buddy">
            <cstring>timeEditDeparture</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEditDeparture"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="lbArrivalDate">
           <property name="text">
            <string>Arrival date</string>
           </property>
           <property name="buddy">
            <cstring>dateEditArrival</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditArrival"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="lbArrivalTime">
           <property name="text">
            <string>Arrival time</string>
           </property>
           <property name="buddy">
            <cstring>timeEditArrival</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEditArrival"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="lbReturnDate">
           <property name="text">
            <string>Return date</string>
           </property>
           <property name="buddy">
            <cstring>dateEditReturn</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QDateEdit" name="dateEditReturn"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="lbReturnTime">
           <property name="text">
            <string>Return time</string>
           </property>
           <property name="buddy">
            <cstring>timeEditReturn</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_9">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QTimeEdit" name="timeEditReturn"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lbDescription">
           <property name="text">
            <string>Description</string>
           </property>
           <property name="buddy">
            <cstring>plainTextEditDescription</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="plainTextEditDescription"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QPushButton" name="pbApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbCansel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbCansel</sender>
   <signal>clicked()</signal>
   <receiver>TrainingForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditDeparture</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateEditArrival</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateEditArrival</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>dateEditReturn</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEditDeparture</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeEditArrival</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeEditArrival</sender>
   <signal>timeChanged(QTime)</signal>
   <receiver>timeEditReturn</receiver>
   <slot>setTime(QTime)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
